lib/IDE/CodeCompletionResultPrinter.cpp

```diff
@@ -41,23 +49,31 @@ void swift::ide::printCodeCompletionResultDescription(
       if (C.is(ChunkKind::TypeAnnotation) ||
           C.is(ChunkKind::CallArgumentClosureType) ||
           C.is(ChunkKind::CallArgumentClosureExpr) ||
+          C.is(ChunkKind::DeclIntroducer) ||
```

Comment:
What was it doing before with `DeclIntroducer`?

Reply:
`DeclIntroducer` is skipped by `getFirstTextChunkIndex()`. Previously, since any attributes from `ASTPrinter` (if exist) were part of `DeclIntroducer`, it used to be skipped. However, now that we have `AttributeAndModifierListBegin` group which may contain modifiers, and is _not_ skipped by `getFirstTextChunkIndex()`. So `DeclIntroducer` may be _after_ `getFirstTextChunkIndex()`.